pkg/api/hosttwin.go

```diff
@@ -0,0 +1,62 @@
+package api
```

Comment:
I appreciate that these methods (and tests) are there to enable the host twin functionality. But from my understanding these are generic. Is not worth adding them to `highLevelApi*` instead? (I get that if they're imported - the end user will just see `api` as the package.

Reply:
I've moved these to high level APi